cxx/distributions/zero_mean_normal.hh

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024
+// See LICENSE.txt
+
+#pragma once
+#include <random>
+#include <tuple>
+#include <variant>
+
+#include "base.hh"
+#include "util_math.hh"
+
+// A normal distribution that is known to have zero mean.
+// This class is not intended to be directly used as a data model, but
+// rather to support the Gaussian emissions model.
+class ZeroMeanNormal : public Distribution<double> {
+ public:
+  // We use an Inverse gamma conjugate prior, so our hyperparameters are
+  double alpha = 1.0;
+  double beta = 1.0;
```

Comment:
Could these be renamed the same was as normal? Admittedly I don't love the single letter naming scheme or alpha and beta since I find these opaque, but would prefer the naming for both to be the same.

Reply:
They could, but I think the most important thing is that they match the math.  Since https://www.cs.ubc.ca/~murphyk/Papers/bayesGauss.pdf and https://j-zin.github.io/files/Conjugate_Bayesian_analysis_of_common_distributions.pdf both use alpha and beta, I would strongly prefer that we keep those as the variable names.